dali/operators/geometry/affine_transforms/transform_shear.cc

```diff
@@ -163,16 +179,16 @@ class TransformShearCPU
   }
 
  private:
-  int InferNumDims(const Argument<std::vector<float>> &arg) {
-    DALI_ENFORCE(arg[0].size() == 2 || arg[0].size() == 6,
+  int InferNumDims(const ArgValue<float, DynamicDimensions> &arg) {
```

Comment:
Is that used after introducing shape_from_size?

Reply:
True. I'll remove